### DISCUSSION TITLE
感觉附件上传的同时应该可以方便的修改名称或者url，这样就能替代图床了

### DISCUSSION THREAD

**Discussion by IshuaWang** [halo-dev/halo#7027] [Ideas]
目前没法再上传的时候方便地修改名称或者url，导致可能还是需要图床的辅助。如果有这个功能，可能就不需要要另建图床了

**Comment by JohnNiang** [comment]
目前，支持在附件管理中修改附件的名称，但是 URL 目前无法做到手动修改。

**Reply by IshuaWang** [reply]
我指的是自动化修改名称，比如附加日期时间这种前缀，手动修改还是麻烦

**Reply by guqing** [reply]
不是很明白你的描述，建议描述一下具体使用场景

s3 插件目前支持上传时自动根据规则重命名附件
<img width="552" alt="image" src="https://github.com/user-attachments/assets/a679751d-c4d0-4b23-adfb-062a97e25e3c">

**Reply by IshuaWang** [reply]
可能我没注意到这个插件，我用的是原生的附件上传功能。就是上传的时候可以重命名，但是有时可能不注意会有重名，如果数量多了很难保证唯一或者容易区分，这时就可能需要附加时间日期这种前缀。但是目前原生的附件上传不能做到。

**Reply by ruibaby** [reply]
我认为是可以为本地附件存储添加命名规则的功能的，甚至为所有存储策略统一提供这个配置。不过目前的问题是默认的本地存储策略不能修改配置（比如之前提供的文件大小限制，文件类型限制在默认的存储策略都不能修改），这一点我觉得会造成体验问题，毕竟要配置这些必须得新建存储策略了。
